modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -39,17 +40,13 @@
     },
     "RequestFromArchive": {
       "Parameters": {
-        "cma": {
-          "event.$": "$",
-          "ReplaceConfig": {
-            "FullMessage": true
-          },
-          "task_config": {
-            "asyncOperationId": "{$.cumulus_meta.asyncOperationId}",
-            "s3MultipartChunksizeMb": "{$.meta.collection.meta.s3MultipartChunksizeMb}",
-            "defaultBucketOverride": "{$.meta.collection.meta.orca.defaultBucketOverride}",
-            "defaultRecoveryTypeOverride": "{$.meta.collection.meta.orca.defaultRecoveryTypeOverride}"
-          }
+        "event.$": "$",
+        "input.$": "$.previous_step_output",
+        "optionalValues": {
+          "config.defaultRecoveryTypeOverride": "event.meta.collection.meta.orca.defaultRecoveryTypeOverride",
```

Comment:
Noticed that all these keys are `config.*` I am wondering if from a structure standpoint why not do a `config` key which results in an object instead of the preface. so instead of what you currently have it would have a structure like this.

```
"optionalValues": {
    "config": {
        "defaultRecoveryTypeOverride": ...
        "defaultBucketOverride": ...
        "s3MultipartChunksizeMb": ...
        "asyncOperationId": ...
    }
}
```
```

Reply:
If you are suggesting we use `"asyncOperationId": "$.cumulus_meta.asyncOperationId"`, AWS raises an error if any part of the path is not present, so we can't just drop them into config.

If you are suggesting we use `"config.asyncOperationId": "event.cumulus_meta.asyncOperationId"` then we'll have no way of knowing which values are placeholders, and which values should be treated as keys.

Reply:
will work with Andrew to update the structure like Brad suggested

Reply:
New structure will be
"optionalValues": {
  "[event key, such as 'input' or 'conifg']": {
    "[more keys if applicable]": {
      "[final key]": "event.meta.collection.meta.orca.defaultRecoveryTypeOverride"
    }
  }
}